tests/test_ddsketch.py

```diff
@@ -65,7 +65,7 @@
 TEST_BIN_LIMIT = 1024
 
 
-class TestDDSketches(ABC):
+class BaseTestDDSketches(ABC):
```

Comment:
This name change is because pytest would pick up the old ABC and try to run it as a test which would fail because the methods were not implemented.